browser/brave_rewards/test/rewards_page_browsertest.cc

```diff
@@ -120,6 +171,43 @@ class RewardsPageBrowserTest : public InProcessBrowserTest {
     LoadScript("test_setup.js");
   }
 
+  using RequestHandlerResult = std::optional<std::pair<int, std::string>>;
+  using RequestHandler =
+      base::RepeatingCallback<RequestHandlerResult(const GURL&,
+                                                   const std::string&)>;
+
+  void SetRequestHandler(RequestHandler handler) {
```

Comment:
Request handling has been moved out of the test scripts and into the tests on the C++ side.